src/commands/deploy/latest.js

```diff
@@ -163,11 +163,26 @@ export default async function main(
   }
 
   const { apiUrl, authConfig: { token }, config: { currentTeam } } = ctx;
-
   const { log, debug, error, print } = output;
   const paths = Object.keys(stats);
   const debugEnabled = argv['--debug'];
 
+  if (argv['--name']) {
```

Comment:
May be we can deprecate and warn instead of throwing errors. What do you think?

Reply:
@arunoda why deprecate?

Reply:
@rauchg because this will throw an error when a user used it.
I think we can keep simply remove it.

But we should try to deprecate the `.name` field and warn it instead removing it.
Otherwise, a lot of GH deployments(and local deployments too) will fail after we use this CLI version.
(I'm just looking for options?)

Reply:
Read me comment above – I think we should definitely deprecate and not throw.

Reply:
@leo got it. I'll change it.